apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerDbType.StringParsing.cs

```diff
@@ -199,6 +208,11 @@ public override string ToString()
                 s.Append(">");
                 return s.ToString();
             }
+            if (TypeAnnotationCode == TypeAnnotationCode.PgNumeric)
+            {
+                return "PGNUMERIC";
```

Comment:
Given that there's no type named PGNumeric, I wonder if it'd make more sense to do "NUMERIC{PG}", but that's just a very tiny nit.